controller/User/getWatchList.js

```diff
@@ -1,50 +1,57 @@
 const models = require('../../models');
 var jwt = require('jsonwebtoken')
+/** @description This functions gives list of all movies present in the watchlist
+ * @param {object} req - Request object will be null
+ * @param {object} res -  Reponse object with all the movies list  if success or error message if there is an error.
+ * @param {requestCallback} next - The callback that calls the error handling middleware.
+ *  @returns {Promise}
+*/
 const getWatchList = async (req, res, next) => {
     try {
-        
+
         const token = req.headers['access-token']
```

Comment:
Remove this getting user by using headers in every file. Use appending user to req after Authorization

Reply:
yes i have removed that headers as there is no requirement i havent used them any where